packages/grafana-runtime/src/services/live.ts

```diff
@@ -20,68 +16,47 @@ export interface LiveDataFilter {
 }
 
 /**
- * @alpha
+ * Indicate if the frame is appened or replace
+ *
+ * @public -- but runtime
  */
-export interface LiveDataStreamOptions {
```

Comment:
nit: I know this is marked as alpha but I know that there are plugins using this feature. Should we provide some migration notes to make the transition easier?